app/controllers/discourse_automation/append_last_checked_by_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module DiscourseAutomation
+  class AppendLastCheckedByController < ApplicationController
+    requires_login
+
+    def post_checked
+      post = Post.find(params[:post_id])
+      guardian = Guardian.new(current_user)
```

Comment:
The `ApplicationController` already provides us with a guardian on the current_user

https://github.com/discourse/discourse/blob/c700cf321342a2639b27ed6c9a4fb81205abf2f2/app/controllers/application_controller.rb#L499-L506